### PR TITLE
fix: display the PartitionBound and PartitionDef correctly

### PR DESCRIPTION
**Pull request by Kelvinyu1117**
I hereby agree to the terms of the [GreptimeDB CLA](https://github.com/GreptimeTeam/.github/blob/main/CLA.md).

## Refer to a related PR or issue link (optional)
#3985

## What's changed and what's your intention?

__!!! DO NOT LEAVE THIS BLOCK EMPTY !!!__

Please explain IN DETAIL what the changes are in this PR and why they are needed:

- Summarize your change (**mandatory**)
- How does this PR work? Need a brief introduction for the changed logic (optional)
- Describe clearly one logical change and avoid lazy messages (optional)
- Describe any limitations of the current code (optional)

## Checklist

- [ ] I have written the necessary rustdoc comments.
- [ ] I have added the necessary unit tests and integration tests.
- [ ] This PR requires documentation updates.

### CODE REVIEW

File: src/partition/src/partition.rs [hunk 1]
```diff
@@ -72,8 +72,7 @@ impl Display for PartitionDef {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         write!(
             f,
-            "({}) VALUES LESS THAN ({})",
-            self.partition_columns.iter().join(", "),
+            "VALUES LESS THAN ({})",
```
Comment (killme2008) [inline src/partition/src/partition.rs:75]: The `VALUE LESS THAN` is unnecessary too.
```suggestion
            "{}",
```

**Comment by killme2008** [comment]
@Kelvinyu1117 The sqlness results need updating too.

**Comment by Kelvinyu1117** [comment]
> @Kelvinyu1117 The sqlness results need updating too.

I think `partition.result` is checked in.

**Comment by killme2008** [comment]
> > @Kelvinyu1117 The sqlness results need updating too.
> 
> I think the changes of `partition.result` is checked in.

Yep, but `VALUES LESS THAN` was removed, the results will be changed too.

**Comment by Kelvinyu1117** [comment]
> > > @Kelvinyu1117 The sqlness results need updating too.
> > 
> > 
> > I think the changes of `partition.result` is checked in.
> 
> Yep, but `VALUES LESS THAN` was removed, the results will be changed too.

Oops, updated, thanks.

**Review by killme2008 — APPROVED** [review]
LGTM

**Review by fengjiachun — APPROVED** [review]
LGTM

**Comment by fengjiachun** [comment]
Thank you! @Kelvinyu1117

**Pull request merged**